lib/rules/web/admin_console/4.8/deployment.xyaml

```diff
@@ -149,4 +149,38 @@ select_replica_count_and_selector:
   params:
     text: Replica count and selector
   action: click_checkbox_input_by_text
-
+check_and_click_metrics_tab:
+  elements:
+  - selector:
+      xpath: //*[@id="content-scrollable"]//a[contains(text(),'Metrics')]
+  params:
+    tab_name: Metrics
+  action: click_tab
+check_charts_on_metrics_tab:
```

Comment:
we should try to avoid using `on_somewhere` clause since it should probably change, suggestting `check_workloads_metrics`